app/src/main/java/com/example/tanya/godeltechchallengeandroid/util/Prefs.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.tanya.godeltechchallengeandroid.util
+
+import android.content.SharedPreferences
+import javax.inject.Inject
+
+const val PREFS_NAME = "godeltechchallengeandroid_prefs"
+
+class Prefs
```

Comment:
I don't like the SharedPreferences name. Let's give it the following name "ApplicationPreferences" to reflect the preferences responsibility.